api/middleware/nativeAuth.go

```diff
@@ -109,7 +109,7 @@ func (middleware *nativeAuth) MiddlewareHandlerFunc() gin.HandlerFunc {
 			return
 		}
 
-		c.Set(UserAddressKey, string(authToken.GetAddress()))
+		c.Set(UserAddressKey, "erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th")
```

Comment:
this should be reverted to `c.Set(UserAddressKey, string(authToken.GetAddress()))`

also, in order to skip native auth verification on this endpoint, on api.toml, `{ Name = "/sign-message", Open = true, Auth = true },` should have `Auth = false`. This way you basically whitelist this endpoint for having the Bearer mandatory

Reply:
oh wow. I though I didn't commit this one. sorry